### ISSUE TITLE
Docs mention [alias.nodejs], but needs to be [aliases.nodejs]

### ISSUE THREAD

**Issue by amoosbr** [jdxcode/rtx#9]
This is probably just an WIP issue. Just wanted to mention it, in case someone else wonders as I did.

Instead of using [alias.nodejs] as documented in the [README.md](https://github.com/jdxcode/rtx/blob/main/README.md):
```toml
[alias.nodejs]
my_custom_node = '18'  # makes `rtx install nodejs@my_custom_node` install node-18.x
```
it must be [aliases.nodejs]
```toml
[aliases.nodejs]
my_custom_node = '18'  # makes `rtx install nodejs@my_custom_node` install node-18.x
```

Not sure, what the final idea is. Using [alias.nodejs] and updating the code or using [aliases.nodejs] and updating the README.

**Comment by jdxcode** [comment]
oof this _should_ actually be `alias`, I need to update the code. Fair warning, this would normally require a major version bump but since this project just came out yesterday I'm going to make this change in a patch release. This isn't something I'm likely to do ever again.

**Comment by jdxcode** [comment]
this fix is out in 1.2
